lib/KalturaHelpers.php

```diff
@@ -200,8 +200,8 @@ public static function getOption( $name, $default = null ) {
 		$name    = is_string( $name ) ? $name : null;
 		$default = is_bool( $default ) ? $default : null;
 
-		$value = get_option( $name, $default );
-		if ( ! is_null( $value ) ) {
+		$value = self::isPluginNetworkActivated() ? get_site_option( $name, $default ) : get_option( $name, $default );
```

Comment:
The original code had a fallback to `get_site_option` when `get_option` returned null.
I think that you can keep on using it, it will remove the need for `isPluginNetworkActivated`.

See https://github.com/kaltura/all-in-one-video-pack.wordpress/blob/2a51cef383f1fa02aecefa1b9d0fc2b7adbc7d0f/lib/KalturaHelpers.php#L238